.doc_gen/metadata/cross_metadata.yaml

```diff
@@ -126,11 +126,15 @@ cross_TextractExplorer:
     - sqs
     - textract
 cross_SubmitDataApp:
-  title: Build an app to submit data to an &DDB; table
+  title: Build an application to submit data to an &DDB; table
   title_abbrev: Build an app to submit data to an &DDB; table
```

Comment:
Make this change to both title and title_abbrev:
an &DDB; -> a &DDB; 
(because this renders as "a DynamoDB")

Reply:
fixed